crates/sdk/src/network-v2/mod.rs

```diff
@@ -1,13 +1,20 @@
-pub mod client;
+mod client;
+mod error;
 mod json;
-pub mod prover;
+mod prover;
 mod sign_message;
+mod types;
 #[rustfmt::skip]
-pub mod proto;
+mod proto;
+
+pub use client::*;
```

Comment:
flattening this to make the API clean e.g. `use sp1_sdk::network_v2::{Error, FulfillmentStrategy}`